pkg/web/templates/alerts.gohtml

```diff
@@ -37,9 +37,11 @@
     </tbody>
 </table>
 
-<br /><br /><br /><br /><br /><br /><br />
-<h3 class="filtered">Filtered Alerts</h3>
-<table class="widetable filtered">
+<h3 class="filtered">
+    Filtered Alerts ({{ len .Content.Blocked }})
+    <button id="toggle-filtered-alerts">Show</button>
+</h3>
+<table id="filtered-table" class="widetable filtered hidden">
```

Comment:
Unsure how it interacts with the reload mechanic.  Would it stay open if the backend sends new data?  (new data means that the full `alerts.gohtml` is sent to the frontend and replaces the previous content.

Reply:
Idea: Maybe `turbo:before-render` could be used to toggle it open again on reload.
https://turbo.hotwired.dev/reference/events

I guess.. we could use the same mechanic for-reopening items as well.
There is no way to distinguish rows right now, but for silencing there is an ID available.  We could use `aria-data` or something to make rows accessible via the ID.

Reply:
I will have a look into it :)

Reply:
For now I've made it possible that the state is preserved when the site is reloaded.  It works by simply re-registering the handler after the stream rendered.